infrastructures/openshift/src/test/java/org/eclipse/che/workspace/infrastructure/openshift/environment/OpenShiftEnvironmentFactoryTest.java

```diff
@@ -158,39 +154,34 @@ public void shouldCreateOpenShiftEnvironmentWithPVCsFromRecipe() throws Exceptio
     assertEquals(osEnv.getPersistentVolumeClaims().get("pvc2"), pvc2);
   }
 
-  @Test
-  public void ignoreRoutesWhenRecipeContainsThem() throws Exception {
-    final List<HasMetadata> objects =
-        asList(
-            new RouteBuilder().withNewMetadata().withName("route1").endMetadata().build(),
-            new RouteBuilder().withNewMetadata().withName("route2").endMetadata().build());
-    when(parsedList.getItems()).thenReturn(objects);
+  public void addRoutesWhenRecipeContainsThem() throws Exception {
```

Comment:
Looks like `@Test` is missing and this test method won't be run

Reply:
Fixed -- good catch.